members/members.css

```diff
@@ -0,0 +1,127 @@
+.title {
+    text-align: center;
+    margin-bottom: 2em;
+    font-size: 3.5em;
+}
+
+.sub-title {
+    font-size: 25px;
+    color: #444444;
+    position: relative;
+    padding-bottom: 1.25em;
+}
+
+.sub-title:after{
+    border-bottom: solid 1px;
+    bottom: 0;
+    content: '';
+    display: inline-block;
+    left: 0;
+    position: absolute;
+    width: 3rem;
+}
+
+h1 {
+    font-size: 30px;
+}
+
+h2{
+    font-size: 25px;
+}
+
+h3{
+    font-size: 20px;
+}
+
+h4{
+    font-size: 15px;
+}
+
+/* Five columns side by side */
+.column-card {
+  float: left;
+  width: 20%;
+  padding: 0 8px;
+  margin-bottom: 16px;
+}
+
+/* Display the columns below each other instead of side by side on small screens */
+@media screen and (max-width: 650px) {
+  .column-card {
+    width: 100%;
+    display: block;
+  }
+}
+
+/* avatar label alignment */
+.container-card {
+  padding: 0 16px 16px 16px;
+  margin: 10px 0;
+  text-align: center;
+}
+
+.title-card {
+  color: grey;
+}
+
+.avatar-execs{
```

Comment:
What do you think of changes up these classes to something like so:
```
.clickable {
    cursor: pointer;
}

.clickable:hover {
    opacity: 0.8;
}
```
Then when we are creating the exec avatars we can do something like
```
<img id="{{ item.identification }}-img" src="{{ site.baseurl }}{{ item.image }}" class="avatar clickable">
```

This way, both types of avatars maintain the same base styling properties 😄

Reply:
that is much cleaner, done :)